test/Java9andUp/playlist.xml

```diff
@@ -229,7 +229,40 @@
 		<subsets>
 			<subset>SE90</subset>
 		</subsets>
-	</test>	
+	</test>
+	<test>
+		<testCaseName>StaticAccessChecks-IllegalAccessPermit</testCaseName>
+			<command>$(JAVA_COMMAND) $(JVM_OPTIONS) --illegal-access=permit\
+     -cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(TEST_RESROOT)$(D)GeneralTest.jar$(P)$(LIB_DIR)$(D)asm-all.jar$(Q) \
```

Comment:
Please align `<testCaseName>` and `<command>` in playlist.xml and same in testng.xml